src/transformers/testing_utils.py

```diff
@@ -42,6 +42,7 @@
 from .integrations import (
     is_clearml_available,
     is_fairscale_available,
+    is_optimum_available,
```

Comment:
Import from utils.

Reply:
Fixed in [9c5e0d9](https://github.com/huggingface/transformers/pull/21259/commits/9c5e0d9308b5c4882805058023729fb6dd63cbe9)